model/qti/ImportService.php

```diff
@@ -843,4 +852,30 @@ protected function getMetadataImporter()
         }
         return $this->metadataImporter;
     }
+
+    /**
+     * Retrieve the labels of all parent classes up to base item class.
+     *
+     * Return values: an empty string if the $class parameter is not a class object
+     *                an empty array if the $class parameter is the base item class
+     *                an array in other cases, ordered from base item class to the class given as parameter
+     *
+     * @param mixed $class Class from which to retrieve.
+     * @return array|string
+     */
+    public function retrieveFullPathLabels($class)
```

Comment:
unit test?

Reply:
Looking for the possibility to do that before tomorrow.

Reply:
Added tests for this method.